akka-persistence/src/main/scala/akka/persistence/snapshot/SnapshotStore.scala

```diff
@@ -14,14 +14,16 @@ import akka.persistence._
 /**
  * Abstract snapshot store.
  */
-trait SnapshotStore extends Actor {
+trait SnapshotStore extends Actor with ActorLogging {
   import SnapshotProtocol._
   import context.dispatcher
 
   private val extension = Persistence(context.system)
   private val publish = extension.settings.internal.publishPluginCommands
 
-  final def receive = {
+  final def receive = receiveSnapshotStore orElse receivePluginInternal
```

Comment:
there is a warning here, use:

    receiveSnapshotStore.orElse[Any, Unit](receivePluginInternal)